ambari-server/src/main/java/org/apache/ambari/server/controller/AlertTargetSwagger.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ambari.server.controller;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.ambari.server.controller.internal.AlertGroupResourceProvider;
+import org.apache.ambari.server.controller.internal.AlertTargetResourceProvider;
+
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * Request / response schema for AlertTarget API Swagger documentation generation. The interface only serves documentation
+ * generation purposes, it is not meant to be implemented.
+ */
+public interface AlertTargetSwagger extends ApiModel {
```

Comment:
To more accurately reflect the JSON document, the outer `AlertTarget` map should be modelled, too.

```
"AlertTarget" : {
    "name" : "asdf",
    "notification_type" : "test",
    ...
}
```

Example:

https://github.com/apache/ambari/blob/c3363500db5a4d28490d5429832c3bfe0b55cf79/ambari-server/src/main/java/org/apache/ambari/server/controller/QuickLinksResponse.java#L31-L39